src/WebJobs.Script/Workers/Rpc/WebHostRpcWorkerChannelManager.cs

```diff
@@ -124,7 +124,7 @@ public async Task SpecializeAsync()
 
             if (_workerRuntime != null && rpcWorkerChannel != null)
             {
-                if (UsePlaceholderChannel(_workerRuntime))
+                if (UsePlaceholderChannel(rpcWorkerChannel))
```

Comment:
When `WEBSITE_USE_PLACEHOLDER_DOTNETISOLATED` is not truthy and specialization happens. Don't we need to update the worker config so that the executable path is set to the customer payload dll? Currently it looks like it is still going to use the worker config which points to the FunctionsNetHost.exe.

Reply:
I think I mentioned offline -- but will capture it here as well. Once specialization occurs, the host will load the worker's FunctionMetadataProvider assembly from its payload. This has code to overwrite the configuration for dotnet-isolated to point to the customer's app directly. So, we don't actually need any code here -- it just happens automatically because of this:

https://github.com/Azure/azure-functions-host/blob/6b3e19463055d44cae61cff32c93eb58f0cf790e/src/WebJobs.Script/Workers/Rpc/Configuration/RpcWorkerConfigFactory.cs#L105-L106